youtube_service.py

```
@@ -388,6 +389,47 @@ async def get_video_transcript(self, video_id: str) -> Optional[VideoTranscript]
         
         logger.error(f"All transcript fetch strategies failed for {video_id}")
         return None
+
+    async def _fetch_transcript_timedtext(self, video_id: str) -> Optional[VideoTranscript]:
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)